src/components/rockets/rockets.js

```diff
@@ -0,0 +1,40 @@
+import './rockets.css';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRocketsData, reserveRocket, cancelReserve } from '../../redux/rockets/rocketsSlice';
+
+const Rockets = () => {
+  const rocketsData = useSelector((state) => state.rockets.rocketsLists);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (rocketsData.length === 0) dispatch(fetchRocketsData());
+  }, [dispatch, rocketsData.length]);
+
+  return (
+    <>
+      {rocketsData.map((rocket) => (
+        <div className="rockets-container" key={rocket.id}>
+          <img src={rocket.flickr_images} className="rocket-image" alt={rocket.id} />
+          <div className="rocket-data">
+            <p className="rocket-title">{rocket.rocket_name}</p>
+            <p className="rocket-description">
+              {rocket.reserved === true ? <span className="reserved">Reserved</span> : ''}
+              {rocket.description}
+            </p>
+            { rocket.reserved === false
+              ? (
```

Comment:
[OPTIONAL]
- [ ] To make it looks more professional, instead of saying 
```jsx
{rocket.reserved === true ?
```
You might say:
```jsx
{rocket.reserved ?      // means if this value is truthy
```
And 
```jsx
{!rocket.reserved ?
```
Instead of
```jsx
{rocket.reserved === false
```